lib/get-status.js

```diff
@@ -4,18 +4,21 @@
  * @returns {string}
  */
 module.exports = function getStatus (checkSuites) {
+  const numOfTests = checkSuites.length
+
   if (checkSuites.every(suite => suite.conclusion === 'success')) {
-    return 'passing'
+    return numOfTests + '/' + numOfTests + ' passing'
   }
 
   const failures = ['canceled', 'timed_out', 'failed']
-  if (checkSuites.some(suite => failures.includes(suite.conclusion))) {
-    return 'failing'
+  const numOfFailing = checkSuites.filter(suite => failures.includes(suite.conclusion)).length
+  if (numOfFailing > 0) {
+    return numOfFailing + '/' + numOfTests + ' failing'
   }
 
   if (checkSuites.every(suite => suite.conclusion === 'neutral')) {
-    return 'neutral'
+    return numOfTests + '/' + numOfTests + ' neutral'
   }
 
-  return 'mixed'
+  return 'unknown'
```

Comment:
`mixed` is more correct than `unknown` - at this stage in the conditional logic, we _know_ what the sum statuses are, it's just difficult to describe. It might be better to say something like:

```suggestion
  return `${numOfTests}/${numOfTests} mixed`
```